web/src/lib/name-wranglers.test.ts

```diff
@@ -0,0 +1,11 @@
+import { describe, expect, it } from 'vitest';
+import { getInitials } from './name-wranglers';
+
+describe('getInitials', () => {
+	it("converts 'JohnSmith' to 'J'", () => expect(getInitials('JohnSmith')).toBe('J'));
+	it("converts 'John Smith' to 'JS'", () => expect(getInitials('John Smith')).toBe('JS'));
+	it("converts 'john' to 'j'", () => expect(getInitials('john')).toBe('j'));
+	it("converts '风生水起' to '风'", () => expect(getInitials('风生水起')).toBe('风'));
+	it("only returns at most 2 characters", () => expect(getInitials('John Smith The Third')).toBe('JS'));
+	it("doesn't handle emojis well (which is fine)", () => expect(getInitials('😻')).not.toBe('😻'));
```

Comment:
Emojis are just UTF-8, so something like `\u80fe`, which breaks down to, apparently, several code characters. There's not really a good way to tell where an emoji starts or ends in a string afaik.

To make things more complicated, the way something like skin-tones work is through a sequence of emojis, e.g. 🏻👍 = thumbs up with white-1 skin tone. So that `getInitials` you were thinking of would just end up returning 👍, unless we start implementing `getInitials` to wholly parse emoji sequences according to the spec.

So I just opted for something simple here.